src/Bicep.Core.Samples/Files/Parameters_LF/main.bicep

```diff
@@ -73,6 +73,13 @@ param secretObject object
 ])
 param storageSku string
 
+@allowed([
+  1
+  2
+  3
+])
+param intEnum int
+
```

Comment:
This was a manual change to this file (to ensure that `@allowed` decorators could be used to assign a union of integer literals type to parameters.